src/WebJobs.Extensions.CosmosDB/Config/CosmosDBWebJobsBuilderExtensions.cs

```diff
@@ -11,40 +12,33 @@ namespace Microsoft.Extensions.Hosting
     /// <summary>
     /// Extension methods for CosmosDB integration.
     /// </summary>
-    public static class CosmosDBHostBuilderExtensions
+    public static class CosmosDBWebJobsBuilderExtensions
```

Comment:
I just updated all the SDK core extension names as well https://github.com/Azure/azure-webjobs-sdk-extensions/pull/452